packages/typespec-autorest-canonical/src/openapi.ts

```diff
@@ -145,6 +148,7 @@ import {
   Refable,
 } from "./types.js";
 import { AutorestCanonicalEmitterContext, resolveOperationId } from "./utils.js";
+import { getRef } from "./decorators.js";
```

Comment:
Have you tested this? This will not work either no? its not going to get the value from anywhere

Reply:
you should call the `resolveRef` from autorest emitter

Reply:
import it from autorest